tests/integration/test_s3.py

```diff
@@ -252,6 +252,12 @@ def test_can_get_and_put_object(self):
             Bucket=self.bucket_name, Key='foobarbaz')
         self.assertEqual(data['Body'].read().decode('utf-8'), 'body contents')
 
+    def test_can_put_large_string_body_on_new_bucket(self):
+        body = '*' * (5 * (1024 ** 2))
+        response = self.client.put_object(
+            Bucket=self.bucket_name, Key='foo', Body=body)
+        self.assertEqual(response['ResponseMetadata']['HTTPStatusCode'], 200)
```

Comment:
They seem identical except for the body content correct?

If that's the case, can we just pull up the test helper into a base class?